reviewrot/gitlabstack.py

```diff
@@ -0,0 +1,170 @@
+                project = gl.projects.get(os.path.join(user_name, repo_name))
+            except GitlabGetError:
+                log.exception('Project %s not found for user %s',
+                              repo_name, user_name)
+                raise Exception('Project %s not found for user %s'
+                                % (repo_name, user_name))
+            # get merge requests for specified username and project name
+            res = self.get_reviews(uname=user_name, project=project,
+                                   state_=state_, value=value,
+                                   duration=duration)
+
+        else:
+            # get user object
+            groups = gl.groups.search(user_name)
+            if not groups:
+                log.debug('Invalid user/group name: %s', user_name)
+                raise Exception('Invalid user/group name: %s' % user_name)
+
+            # get merge requests for all projects for specified group
+            for group in groups:
+                projects = gl.group_projects.list(group_id=group.id)
+                if not projects:
+                    log.debug("No projects found for user/group name %s",
+                              user_name)
+                for project in projects:
+                    res = self.get_reviews(uname=user_name, project=project,
+                                           state_=state_, value=value,
+                                           duration=duration)
+                    response.append(res)
+        return response
+
+    def get_reviews(self, uname, project, state_=None,
+                    value=None, duration=None):
+        """
+        Fetches merge requests for specified username(groupname)
+        and repo(project) name.
+        Formats the merge requests details and print it on console.
+
+        Args:
+            user_name (str): Gitlab namespace
+            repo_name (str): Gitlab project name for specified
+                             namespace
+            state_ (str): The state for pull requests, e.g, older
+                        or newer
+            value (str): The value in terms of duration for requests
+                         to be older or newer than
+            duration (str): The duration in terms of period(year, month,
+                            hour, minute) for requests to be older or
+                            newer than.
+
+        Returns:
+            res_ (list): Returns list of pull requests for specified
+                         user(group) name and project name
+        """
+        log.debug('Looking for merge requests for %s -> %s',
+                  uname, project.name)
+
+        # get list of open merge requests for a given repository(project)
+        merge_requests = project.mergerequests.list(project_id=project.id,
+                                                    state='opened')
+        if not merge_requests == 0:
```

Comment:
Yeah I was just changing it from "if len(merge_requests) == 0" to "if not merge_requests". Forgot to take out '== 0' part in rush. Thanks